tests/alchemy_test.py

```diff
@@ -0,0 +1,69 @@
+from cdispyutils.log import get_logger
```

Comment:
I would rename this file so it's clear this is for import and not to actually run any tests